File: IntelliJ IDEA/Java/src/demo/comboBoxBindCollection/ViewModel.java
package demo.comboBoxBindCollection;

import com.eruru.mvvm.MVVMObservableCollection;

public class ViewModel {

	private MVVMObservableCollection<String> items = new MVVMObservableCollection<String> () {{
		add ("A");
		add ("B");
		add ("C");
	}};

	public MVVMObservableCollection<String> getItems () {
		return items;
	}

	public void setItems (MVVMObservableCollection<String> items) {
		this.items = items;
	}

}

File: Unity/Assets/Eruru.MVVM/Demos/Test/ViewModel.cs
using UnityEngine;
using System.Collections;
using System.ComponentModel;

namespace Eruru.MVVM.Demo.Test {

	public class ViewModel : INotifyPropertyChanged {

		public event PropertyChangedEventHandler PropertyChanged;
		public string Text {

			get {
				return _Text;
			}

			set {
				_Text = value;
				this.RaisePropertyChanged ();
			}

		}

		string _Text = null;

	}

}